### DISCUSSION TITLE
NES Cart Connector Alternatives

### DISCUSSION THREAD

**Discussion by redchairlabs** [sanni/cartreader#969] [Hardware]
Has anyone sourced a NES  cart connector that was of better quality then the one linked to at aliexpress?

EDIT: The link I referred to is [HERE](https://github.com/sanni/cartreader/wiki/What-to-order#cartridge-slots)

**Comment by sanni** [comment]
You could in theory buy a generic card edge connector with 2.54mm spacing, salvage the pins and design and 3d print a custom 2.5mm cartridge slot to put the pins in. 
I think I saw something like that on Twitter once. Similar to this: https://x.com/Time2Retro/status/1797297940798533685

**Comment by redchairlabs** [comment]
thats actually a good idea. I am unfortunately 3d printing challenged. however, i am fluent in 3D modeling and CAD so maybe i pick this challenge up.

I used to use Fusion 360 before my license ran out. what is everyone using for CAD/CAM these days?
